### PR TITLE
Add camera view

### PR DESCRIPTION
**Pull request by bijington**
Provides the documentation for the new `CameraView` based on PR at https://github.com/CommunityToolkit/Maui/pull/1758

### CODE REVIEW

**Review by jfversluis — COMMENTED** [review]
Couple of minor things

File: docs/maui/views/camera-view.md [hunk 1]
```diff
@@ -0,0 +1,536 @@
+---
+title: CameraView - .NET MAUI Community Toolkit
+author: bijington
+description: "The CameraView provides the ability to connect to a camera, display a preview from the camera and take photos."
```
[the remark this answers is not in the document]
Comment (bijington) [reply docs/maui/views/camera-view.md:4]: Unless I've missed something I believe we only support image capture in v1

[the remark this answers is not in the document]
Comment (jfversluis) [reply docs/maui/views/camera-view.md:4]: In that case we should remove "recording videos" on line 10 😄

File: docs/maui/views/camera-view.md [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+ms.date: 05/23/2024
+---
+
+# CameraView
+
+The `CameraView` provides the ability to connect to a camera, display a preview from the camera and take photos. The `CameraView` also offers the options you would expect to support taking photos and recording videos such as turning the flash on or off, saving the captured media to a file, and offering different hooks for events.
```
Comment (bijington) [inline docs/maui/views/camera-view.md:10]:
```suggestion
The `CameraView` provides the ability to connect to a camera, display a preview from the camera and take photos. The `CameraView` also offers features to support taking photos, controlling the flash, saving captured media to a file, and offering different hooks for events.
```

File: docs/maui/views/camera-view.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bijington) [reply docs/maui/views/camera-view.md:4]: How does it look now?

[the remark this answers is not in the document]
Comment (jfversluis) [reply docs/maui/views/camera-view.md:4]: All good! Thanks!

File: docs/maui/get-started.md [hunk 3]
```diff
@@ -144,6 +144,33 @@ builder
 
 To use the features of the toolkit please refer to the documentation pages for each specific feature.
 
+### [CommunityToolkit.Maui.CameraView](#tab/CommunityToolkitMauiCameraView)
+
+This package enables you to connect to a camera, display a preview from the camera and take photos.
+
+**Package name:** `CommunityToolkit.Maui.CameraView`
```
Comment (brminnick) [inline docs/maui/get-started.md:151]:
```suggestion
**Package name:** `CommunityToolkit.Maui.Camera`
```

I've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.

File: docs/maui/get-started.md [hunk 4]
```diff
@@ -144,6 +144,33 @@ builder
...
+
+**Package url:** https://www.nuget.org/packages/CommunityToolkit.Maui.CameraView
```
Comment (brminnick) [inline docs/maui/get-started.md:153]:
```suggestion
**Package url:** https://www.nuget.org/packages/CommunityToolkit.Maui.Camera
```

I've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.

File: docs/maui/get-started.md [hunk 5]
```diff
@@ -144,6 +144,33 @@ builder
...
+
+#### Initializing the package
+
+First the using statement needs to be added to the top of your *MauiProgram.cs* file
+
+```csharp
+using CommunityToolkit.Maui;
+```
+
+In order to use the `CameraView` correctly, the `UseMauiCommunityToolkitCameraView` method must be called on the `MauiAppBuilder` class when bootstrapping an application the *MauiProgram.cs* file. The following example shows how to perform this.
```
Comment (brminnick) [inline docs/maui/get-started.md:163]:
```suggestion
In order to use the `CameraView` correctly, the `UseMauiCommunityToolkitCamera()` method must be called on the `MauiAppBuilder` class when bootstrapping an application the *MauiProgram.cs* file. The following example shows how to perform this.
```

I've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.

File: docs/maui/get-started.md [hunk 6]
```diff
@@ -144,6 +144,33 @@ builder
...
+
+```csharp
+var builder = MauiApp.CreateBuilder();
+builder
+    .UseMauiApp<App>()
+    .UseMauiCommunityToolkitCameraView()
```
Comment (brminnick) [inline docs/maui/get-started.md:169]:
```suggestion
    .UseMauiCommunityToolkitCamera()
```

I've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.

**Review by brminnick — CHANGES_REQUESTED** [review]
Thanks Shaun! II've updated the NuGet package to `CommunityToolkit.Maui.Camera` to allow us to add future camera features in the future.

I've submitted the code changes here in this review.

**Review by jfversluis — APPROVED** [review]
Welcome to the .NET MAUI Community Toolkit CameraView!

**Pull request merged**